src/config/makeWebpackConfig.ts

```diff
@@ -78,29 +56,20 @@ export default function makeWebpackConfig({
         },
       },
       minimizer: [
-        new TerserPlugin({
-          parallel: true,
-          terserOptions: {
-            ie8: false,
-            output: {
-              comments: false,
-            },
-          },
-        }),
+        '...',
         // @ts-ignore: Appears that the library might have incorrect definitions
         new CssoWebpackPlugin({ restructure: false }),
       ],
     },
     plugins: [
-      new webpack.IgnorePlugin(/^electron$/),
+      new webpack.IgnorePlugin({ resourceRegExp: /^electron$/ }),
       new VueLoaderPlugin(),
       new MiniCssExtractPlugin({
         // Options similar to the same options in webpackOptions.output
         // both options are optional
         filename: '[name].bundle.css',
         chunkFilename: '[id].bundle.css',
       }),
-      ...(debug ? [new WriteFilePlugin()] : []),
```

Comment:
i don't fully understand what this was doing, another for @pastelsky. it was writing the output to disk during watch mode i guess but to what end? the plugin doesn't support webpack 5 so im wondering/hoping we can drop it